deploy.sample.php

```diff
@@ -45,6 +43,16 @@
     'var/versions',
     'var/sessions'
 ]);
+// the configuration files of pimcore that will be processed at creation
+set('pimcore_shared_configurations', [
+    'var/config/website-settings.php',
+    'var/config/reports.php',
+    'var/config/extensions.php',
+    'var/config/web2print.php',
+    'var/config/workflowmanagement.php',
+    'var/config/perspectives.php',
+    'var/config/customviews.php'
```

Comment:
this shouldn't be there as well, causes an pimcore failure when it returns empty array

Reply:
same for perspectives.... doesn't show any tree views (documents, assets, objects) when empty